apps/08jquery-towers-of-hanoi/script.js

```diff
@@ -1,5 +1,37 @@
 'use strict';
 
+var $block = null;
 $(document).ready(function() {
-  // Put app logic here
+  $('[data-stack]').on('click', function() {
+
+    if ($block === null) {
+
+      $block = $(this).children().last().detach();
+
+    } else {
+      if (
+        // IF value of data-block attribute of block is less than that of last
```

Comment:
This is a long comment block, but I like it. Helpful when you come back to the code much later